client/my-sites/backup/backup-date-picker/test/index.jsx

```diff
@@ -55,206 +59,184 @@ describe( 'BackupDatePicker', () => {
 	// --- DATE DISPLAY ---
 
 	test( "Shows 'Yesterday' as the previous date if the current date is today", () => {
-		const today = moment();
-
-		const picker = shallow( <BackupDatePicker selectedDate={ today } onDateChange={ () => {} } /> );
+		Date.now = jest.fn().mockReturnValue( new Date( '2020-12-04T12:00:00.000Z' ) );
```

Comment:
This will work, but can't we just mock `moment`, or even just provide a specific date to `moment()`? Seems like that would be a clearer way to get there.

Reply:
Hi @tyxla, we have mocked `Date.now` so that the current date calculation is always static and doesn't change (same reason you stated in your last review), we can pass the selected date as `moment( '2020-12-04 12:00 +0000' );` but the current date(which we calculate inside the component) will still be 2022, similarly we can mock moment instead of `Date.now` but that would also mock all moment calls anywhere in the component i.e `moment()` will be mocked to the same result and also `moment('2021-10-11')` for which we shouldn't actually, we can write moment mock function and return the date based on the argument, but that would make things complex I believe.

Reply:
Makes sense to me, thanks for double-checking and elaborating, @vikneshwar 👍